guide/message_transformation/message_transformation.bal

```diff
@@ -0,0 +1,370 @@
+                            res.statusCode = 500;
+                            res.setPayload(untaint err.message);
+                            caller->respond(res) but { error e =>
+                            log:printError("Error sending response", err = e) };
+                        }
+                    }
+                }
+                else {
+                    error err = { message: "Student ID: " + Idvalue + " is not found" };
+                    http:Response res = new;
+                    res.statusCode = 500;
+                    res.setPayload(untaint err.message);
+                    caller->respond(res) but { error e =>
+                    log:printError("Error sending response", err = e) };
+                }
+            }
+            error err => {
+                http:Response res = new;
+                res.statusCode = 500;
+                res.setPayload(untaint err.message);
+                caller->respond(res) but { error e =>
+                log:printError("Error while content reading", err = e) };
+            }
+        }
+
+    }
+}
+
+//The content enricher service
+service<http:Service> enricher bind contentenricherEP {
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/"
+    }
+    enricher(endpoint caller, http:Request validatedreq) {
+        http:Request enrichedreq = validatedreq;
+        var jsonMsg = validatedreq.getJsonPayload();
+        match jsonMsg {
+            json msg => {
+                //get the student's ID value
+                int Idvalue = check <int>msg["id"];
+                //select details from the data table according to the student's ID
+                var selectRet = testDB->select("SELECT * FROM student", ());
+                table dt;
+                match selectRet {
+                    table tableReturned => dt = tableReturned;
+                    error e => io:println("Select data from student table failed: "
+                            + e.message);
+                }
+                //convert the details to a jason file
+                io:println("\nConvert the table into json");
+                var jsonConversionRet = <json>dt;
+                match jsonConversionRet {
+                    json jsonRes => {
+                        //set student's details to the global variable
+                        payload1= untaint jsonRes;
+                        io:println(payload1);
+                    }
+                    error e => io:println("Error in student table to json conversion");
+                }
+                //drop the student details table
+                var ret = testDB->update("DROP TABLE student");
+                handleUpdate(ret, "Drop table student");
+                //select student's results from the student results data table, according to the student's ID
+                var selectRet1 = testDB1->select("select Com_Maths,Physics,Chemistry from StudentDetails where ID = ?", (), Idvalue);
```

Comment:
Shall we rename the SQL keywords with ALL CAPS?

Reply:
Done